libs/api/repository/src/lib/gn4/organizations/organizations-from-metadata.service.ts

```diff
@@ -152,59 +163,69 @@ export class OrganizationsFromMetadataService
     }
   }
 
-  private getAggregationSearchRequest(gnVersion: string) {
+  private getAggregationSearchRequest(
+    gnVersion: string,
+    configFilters?: FieldFilters
+  ) {
     const semVersion = valid(coerce(gnVersion))
-    return this.esService.getSearchRequestBody({
```

Comment:
I don't get why github marks that many changes, while there is only one additional line, weird.